docs/manuals/auth/authorization.md

```diff
@@ -0,0 +1,254 @@
+expected string at the end of the journey.
+
+Note that is a user has both the "exec" and the "sales" roles (the CSO, COO
+or CFO might), the filter is built so that this user only gets the Exec level
+information output.
+
+## Role Traits for GUI pages
+
+Taipy GUI Enterprise edition uses role traits to select pages that
+user can display.
+
+The `AuthorizedPage^` class lets you specify a role traits filter where
+both the *success* and *failure* cases must be a page renderer (whether
+a `Markdown^` or an `HTML^` instance - see the
+[section on Page Renderers](../gui/pages.md/#defining-the-page-content) for more
+information).<br/>
+Instead of calling `Gui.add_page()` for the page renderer, you will
+use the same API, providing the defined `AuthorizedPage^` instance.</br>
+When the requested page is an `AuthorizedPage^`, the role traits filter
+is triggered so that the relevant page renderer is used to display the
+page that the user can see and interact with.
+
+The documentation for `AuthorizedPage^` shows an example of how to use
+this functionality.
+
+## Permissions in Taipy Core
+
+You can control the access to the functionalities exposed by the entities of Taipy
+Core.
+
+Taipy Core uses four predefined user roles names that can be assigned to users. Each of these predefined roles provide a different set of capabilities and are described in details below.
+
+!!! note
+    "Caution: If the application relies on the LDAP authentication, note that Taipy does
+    not provide functions to populate the predefined user roles to the user's LDAP server. The user is expected to populate the roles themselves."
+
+### "TAIPY_READER"
+
+- A *reader* ("TAIPY_READER" role) has the permission to view the various entities and
+  related data in a data node.
+- A *reader*, however, cannot create, edit, delete an entity or edit the data in a data
+  node manually. Readers are not permitted to submit tasks, pipelines or scenarios,
+  subscribe or unsubscribe to an execution or cancel, pause or resume any job.
+
+### "TAIPY_EDITOR"
+
+- An *editor* ("TAIPY_EDITOR" role) has the permission to view, create, edit and delete
+  the various entities and related data in data node.
+- An *editor*, however, cannot execute any task, pipeline or scenario, subscribe or
+  unsubscribe to an execution or cancel, pause or resume any job.
+
+### "TAIPY_EXECUTOR"
+
+- An *executor* ("TAIPY_EXECUTOR" role) has the permission to view the various entities,
+  and related data in data node, submit tasks, pipelines or scenarios for execution,
+  subscribe or unsubscribe an execution, and cancel, pause or resume any job.
+- An *executor*, however, cannot create, edit or delete the various entities, and
+  related data in data node.
+
+### "TAIPY_ADMIN"
+
+- An *admin* ("TAIPY_ADMIN" role) is not restricted at all.<br/>
+  An *admin* is able to perform all actions available to other roles with no
+  restrictions.
```

Comment:
This doc is a duplication from index.md. I believe we must remove it from index.md, and keep it here.